query/network_security/use_network_policies.sql

```diff
@@ -0,0 +1,27 @@
+with analysis as (
```

Comment:
Does this query only check the network policy that's activated for an account, or does it check all network policies (even those not activated) for an account?